src/config/tgprojects.js

```diff
@@ -0,0 +1,89 @@
+export default [
+    {
+		title: 'Covalent',
+        names: 'Arek Der-Sarkissian, Daniel Dai, Muthu Palaniappan, Will O\'Brien, Hritik Arya, Catherine Hu, Daniel Lee, Michael Shi',
+		summary: 'We\'re helping teams build better bonds with virtual icebreakers.',
+        category: 'Games & Graphics',
+		image: '/images/techgala/projects/covalent.png',
+        winner: 'Winner!'
```

Comment:
I realize that our spreadsheet didn't specify what type of award these teams won, that's on me.

Mind updating this to 
```suggestion
        winner: 'Best Large Team'
```